shub/apps/base/sitemap.py

```diff
@@ -23,7 +23,8 @@ def location(self, obj):
 class ContainerSitemap(BaseSitemap):
     changefreq = "weekly"
 
-    def lastmod(self, obj):
+    @staticmethod
```

Comment:
Please don't change these to static either. We are following the Django example: https://docs.djangoproject.com/en/3.1/ref/contrib/sitemaps/